docs/deploy.md

```diff
@@ -0,0 +1,18 @@
+# Deploy Stargz Snapshotter
```

Comment:
Please add that we need `containerd > v1.4.2`.

Reply:
Have added